### PR TITLE
some clarifications and fixes for the tutorials

### PR DESCRIPTION
**Pull request by fzimmerm**
Some small grammar fixes and a clarification on the location of the 'Reset to default' button.
Since a tutorial controller has been modified, the changes have also been applied to Webots in https://github.com/omichel/webots/pull/5568

### CODE REVIEW

File: guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md [hunk 1]
```diff
@@ -231,14 +231,14 @@ can only use one controller at a time.
...
-Controllers don't share the same address space, and they can run in different
+Controllers don't share the same address space, and they can run on different
 processor cores.
 
 <!-- -->
 
 > **Note**:
 Other languages than C are available but may require a setup. Please refer to
-the language chapter to setup the other languages (see [this
+the language chapter to setup another languages (see [this
```
Comment (tn12787) [inline guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]: "the other" is actually fine here, "another" doesn't actually make sense

File: guide/tutorial-4-more-about-controllers-20-minutes.md [hunk 2]
```diff
@@ -128,12 +128,10 @@ Write the prototype of the `main()` function as follows:
 >   // initialize the Webots API
 >   wb_robot_init();
 >   // initialize devices
->   // feedback loop
->   while (1) {
+>   // feedback loop, continue until receiving an exit event
```
Comment (stefaniapedrazzi) [inline guide/tutorial-4-more-about-controllers-20-minutes.md]: The changes to the controllers code have to be ported to the webots repository as well.
In this case to the file https://github.com/omichel/webots/blob/master/projects/samples/tutorials/controllers/e-puck_avoid_collision/e-puck_avoid_collision.c#L32

Comment (fzimmerm) [inline guide/tutorial-4-more-about-controllers-20-minutes.md]: I'll do it when I need a break from "porting" webots to Ogre 2.0

**Comment by stefaniapedrazzi** [comment]
@fzimmerm could you please complete this PR?

Note that if your changes also involve a change in the Webots repository, then you have to apply it on the develop branch.

**Comment by fzimmerm** [comment]
👍

File: guide/tutorial-4-more-about-controllers-20-minutes.md [hunk 3]
```diff
@@ -272,13 +268,8 @@ int main(int argc, char **argv)
...
   }
 
-  // feedback loop
-  while (1) {
-    // step simulation
-    int delay = wb_robot_step(TIME_STEP);
-    if (delay == -1) // exit event from Webots
-      break;
-
+  // feedback loop: step simulation until receiving an exit event
```
Comment (tn12787) [inline guide/tutorial-4-more-about-controllers-20-minutes.md]: receiving an exit event -> an exit event is received

**Comment by fzimmerm** [comment]
👍

**Pull request merged**